cmd/minishift/cmd/start_preflight.go

```diff
@@ -601,14 +614,20 @@ func validateOpenshiftVersion() bool {
 func checkOriginRelease() bool {
 	client := github.Client()
 	ctx := context.Background()
-	_, _, err := client.Repositories.GetReleaseByTag(ctx, "openshift", "origin", viper.GetString(configCmd.OpenshiftVersion.Name))
+
+	requestedOpenShiftVersion, err := getOpenShiftVersion()
```

Comment:
same here better to change the function signature `checkOriginRelease(openshiftVersion string) bool`

Reply:
@praveenkumar `checkOriginRelease()` is a callback function. 
See https://github.com/minishift/minishift/blob/90fa2ae102dbc624acbb31870d3a2e83aca98461/cmd/minishift/cmd/start_preflight.go#L74.

Can't call with arguments.